docs/proposals/20220126_k1-for-devs.md

```diff
@@ -0,0 +1,76 @@
+# KubeOne for Developers
+
+## Goals
+* "Developer Batteries included"
+* Developers should get enabled to start working on the Kubernetes Cluster without having to deal with infrastructure setup problems
```

Comment:
This is unclear and should be clarified.

It's just impossible to get a cluster on any cloud provider without infrastructure setup problems. You at least have to create an account, and in many cases, set up your account, configure billing, configure initial subnets and network gateways (especially on AWS), etc...

Therefore, what are the problems that we should solve?

Reply:
I think this is a different problem you are talking. Yes, you are right you need that stuff to get your cluster via KubeOne up and running. 

The problem which we want to solve with this proposal is more about: OK, now I have my running cluster via KubeOne, what do I need to do to have a cool developer experience on it. By developer I mean the "enduser-developer", the one writing apps on the already existing Kubernetes Cluster.